src/GoogleFitOnFhir/Clients/GoogleFit/Telemetry/GoogleFitExceptionTelemetryProcessor.cs

```diff
@@ -0,0 +1,27 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using Microsoft.Health.Logging.Telemetry;
+
+namespace GoogleFitOnFhir.Clients.GoogleFit.Telemetry
+{
+    public class GoogleFitExceptionTelemetryProcessor : ExceptionTelemetryProcessor
+    {
+        public GoogleFitExceptionTelemetryProcessor()
+            : base(typeof(AggregateException))
+        {
+        }
+
+        public override bool HandleException(Exception ex, ITelemetryLogger logger)
+        {
+            EnsureArg.IsNotNull(ex, nameof(ex));
+            EnsureArg.IsNotNull(logger, nameof(logger));
+
+            return HandleException(ex, logger);
```

Comment:
Is your intention to call the base class here? If so, I think it would be a good idea to make the call more explicate:

```c#
return base.HandleException(ex, logger);
```

Reply:
The example in the other repo has it called this way.  I wasn't being intentional about it, per se.

Reply:
Interesting... I guess we can leave this alone for now.

Reply:
Actually, looking at the example again, I see that the method signature is different.  It is calling the base class version, with more params.  The other params are optional, and at this time we don't have anything equivalent to pass along for those.  Making the call explicit to the base class method is the right approach.